app/User.php

```diff
@@ -244,6 +244,49 @@ public function getGreetingAndName()
         return $this->getGreeting() . ' ' . $this->getFullName();
     }
 
+    /**
+     * Get the user's record by Billet ID. For use in the retrieval of
+     * Space Lords and heads of branches, where a single user holds the
+     * specified billet.
+     *
+     * @param string $billet_id
+     *
+     * @return User|null
+     */
+    public static function getByBilletId(string $billet_id)
+    {
+        $billet_user = null;
+
+        $billet = Billet::where('_id', $billet_id)->first();
+
+        if (isset($billet) && isset($billet['billet_name'])) {
+            $billet_user = User::where('assignment.billet', '=', $billet['billet_name'])
```

Comment:
@cinnion If you're doing equality, you can leave out the comparison operator, and just have the fieldname and value.

If you do that, it's a single item to check.  If you get back the correct class, then the query ran successfully and you have a billet